Library/Formula/arrayfire.rb

```diff
@@ -0,0 +1,45 @@
+class Arrayfire < Formula
+  homepage "http://arrayfire.com"
+  url "https://github.com/arrayfire/arrayfire/archive/v3.0beta.tar.gz"
+  version "3.0beta"
+  sha256 "3d80bd060b2a03e808757f7c2df3035ea357e3476a2cb69af4a3aeae826dd6af"
+
+  depends_on "cmake" => :build
+  depends_on "boost" => :build
+  depends_on "boost-compute" => :build
+  depends_on "pkg-config" => :build
+  depends_on "freeimage"
+  depends_on "fftw"
+  depends_on "clblas"
+  depends_on "clfft"
+
+  patch do
+    url "https://github.com/arrayfire/arrayfire/pull/454.diff"
+    sha256 "cf862821a88ec979d146a9acf34728ef294ec658ed29c1d0adb7af4a5c7bed69"
+  end
```

Comment:
Can you please add an explanation of why the patch is needed in a comment in the formula file. Thanks!